app/scripts/lib/crypto/recovery-keys.js

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * This file contains utilities to help create, bundle and unbundle encrypted
+ * recovery key data.
+ *
+ * For more encryption details, check out
+ + https://github.com/mozilla/fxa-auth-server/blob/master/docs/recovery_keys.md
+ */
+const Base32 = require('./base32');
+
+function importFxaCryptoDeriver() {
+  return import(/* webpackChunkName: "fxaCryptoDeriver" */ 'fxaCryptoDeriver');
+}
+
+function required(object, name) {
+  if (! object) {
+    throw new Error(`Missing ${name}`);
+  }
+}
+
+const RECOVERY_KEY_VERSION = '1';
```

Comment:
There's a slight risk here of using one of the confusable characters as our version identifier.  Supposing we one day want to branch based on the first character, do we have to check whether the user accidentally input it as an "i" or an "l" or similar?  Our future selves might thanks us for using something like "A" that the user is slightly more likely to transcribe reliably.

Reply:
> Our future selves might thanks us for using something like "A"

Interesting!